corehq/apps/locations/tests/test_location_fixtures.py

```diff
@@ -1,28 +1,97 @@
-from django.test import SimpleTestCase
+from datetime import datetime, timedelta
+from django.test import TestCase
 from corehq.apps.locations.models import SQLLocation, LocationType
-from ..fixtures import _location_to_fixture, _location_footprint
+from casexml.apps.phone.models import SyncLog
+from corehq.apps.users.models import CouchUser, CommCareUser
+from ..fixtures import _location_to_fixture, _location_footprint, should_sync_locations, fixture_last_modified
+from corehq.apps.fixtures.models import UserFixtureStatus, UserFixtureType
 
 
-class LocationFixturesTest(SimpleTestCase):
-
-    def test_metadata(self):
-        state = LocationType(
-            domain="test-domain",
+class LocationFixturesTest(TestCase):
+    def setUp(self):
```

Comment:
if you don't need these cleared between tests you can use `setUpClass` and `tearDownClass` which perform a bit better